### PR TITLE
WIP: add ckanext-harvest

### PR DESCRIPTION
**Pull request by avdata99**
About [#1408](https://github.com/GSA/datagov-deploy/issues/1408) 

 - ckanext-harvest upstream added (version 1.1.4 because master and 1.2.0 fails with ckan 2.8)
 - Upgrade pip
 - tested locally: don't start automatically but it's working if I run harvester manually

image

Note: Tests fails since I'm usin [catalog-db](https://github.com/avdata99/catalog-db/pull/1) and [catalog-solr](https://github.com/avdata99/catalog-solr/pull/1) from updates not yet validated.
PR over _update_test_bats_ branch.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,101 +1,5 @@
-#ckan extensions
```
Comment (akariv) [inline requirements.txt:1]: Why did you delete this entire file?

Comment (avdata99) [reply requirements.txt:1]: It's not fully deleted. Since we remove all extensions (to start adding it one by one) I just restart the file with ckanext-harvest only.
Base docker container from OKF already include basic requirements from CKAN @akariv.

Comment (akariv) [reply requirements.txt:1]: I see it also contains pinned dependencies for ckan and the extensions, not just the extensions themselves.

Comment (avdata99) [reply requirements.txt:1]: The `openknowledge/ckan-base:2.8` base image we are using already includes the requirements file from CKAN 2.8. That's why we don't need this reqs here. This branch run CKAN and also is able to harvest.

We can also go an step back and use the base container in which we [manage requirements](https://github.com/okfn/docker-ckan/blob/master/ckan-base/2.8/Dockerfile#L66) also for ckan.

**Comment by avdata99** [comment]
Closed since this not required
